crates/primitives/src/env.rs

```diff
@@ -424,10 +424,10 @@ impl Env {
     ///
     /// [EIP-4844]: https://eips.ethereum.org/EIPS/eip-4844
     #[inline]
-    pub fn calc_data_fee(&self) -> Option<u64> {
+    pub fn calc_data_fee(&self) -> Option<u128> {
```

Comment:
```suggestion
    pub fn calc_data_fee(&self) -> Option<U256> {
```

revm balance is represented as U256 so it would be good to return same type.